assets/src/dashboard/app/api/useSettingsApi.js

```diff
@@ -76,20 +76,39 @@ export default function useSettingsApi(
   }, [dataAdapter, globalStoriesSettingsApi]);
 
   const updateSettings = useCallback(
-    async ({ googleAnalyticsId }) => {
+    async ({
+      googleAnalyticsId,
+      publisherLogoIds,
+      publisherLogoIdToRemove,
+    }) => {
       try {
+        const query = {};
+        if (googleAnalyticsId) {
+          query.web_stories_ga_tracking_id = googleAnalyticsId;
+        }
+
+        if (publisherLogoIds) {
+          query.web_stories_publisher_logos = [
+            ...new Set([...state.publisherLogoIds, ...publisherLogoIds]),
+          ];
```

Comment:
When an update to the publisher logos on settings is requested, we have to signal the entire list of updates since the same key is used for removing logos. 

were we to update with just the new one(s) then the existing publisher logo ids would be replace.d